src/main/java/com/borchowiec/coronavirustrackerforpoland/service/GraphDataType.java

```diff
@@ -0,0 +1,5 @@
+package com.borchowiec.coronavirustrackerforpoland.service;
+
+public enum GraphDataType {
+    CONFIRMED, DEATHS, RECOVERIES, NEW_CONFIRMED, NEW_DEATHS, NEW_RECOVERIES
```

Comment:
In Polish it means "przeprowadzone testy". It could make fraction: confirmed cases / conducted test to make a graph of accuracy of countrywide testing (will send u link with description how it works)

Reply:
Ok i got it. So far our external api doesn't offer this type of data. So maybe in future but not in this feature i guess.

Reply:
By saying "conducted tests" I mean number of all test that have been done since we have epidemic. It could make new graph: confirmed cases to tests ratio (something similar is used to judge quality of healthcare)